### PR TITLE
chore: add instruction detail in _auth.py print statement. Fixes #4987

### LINKED ISSUE #4987: Add clear instructions for authorization code / token
I was following [this official example](https://colab.research.google.com/github/kubeflow/website/blob/master/content/en/docs/pipelines/sdk/python-function-components.ipynb) to submit a pipeline via Jupyter or Python script

I filled in the client id and credentials [following these instructions](https://www.kubeflow.org/docs/gke/pipelines/authentication-sdk/#connecting-to-kubeflow-pipelines-in-a-full-kubeflow-deployment), but when I try to [compile the pipeline](https://www.kubeflow.org/docs/pipelines/sdk/build-component/#compile-the-pipeline) with
```
dsl-compile --py [path/to/python/file] --output [path/to/output/tar.gz]
```
The resulting prompt doesn't make it evident that the url can be copy pasted into the browser, to get the Authorization code, and I got stuck on it for a while. Looking back it's kind of an Occam's razor thing, but it could have saved a lot of time with even simple instructions "copy paste this url"

Screen Shot 2021-01-12 at 5 46 15 PM copy

### PR DESCRIPTION
**Pull request by susan-shu-c**
**Description of your changes:**

Suggested fix for #4987

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

File: sdk/python/kfp/_auth.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ def get_auth_code(client_id):
     auth_url = "https://accounts.google.com/o/oauth2/v2/auth?client_id=%s&response_type=code&scope=openid%%20email&access_type=offline&redirect_uri=urn:ietf:wg:oauth:2.0:oob"%client_id
     print(auth_url)
     open_new_tab(auth_url)
-    return input("Authorization code: ")
+    return input("The above url will direct you to a browser to retrieve the authorization code. Input authorization code: ")
```
Comment (numerology) [inline sdk/python/kfp/_auth.py:181]:
```suggestion
    return input("If there's no browser window prompt, please direct to the URL above, then copy and paste the authorization code here: ")
```

**Review by numerology — COMMENTED** [review]
Thanks! @susan-shu-c

File: sdk/python/kfp/_auth.py (hunk 1 above)
Comment (susan-shu-c) [reply sdk/python/kfp/_auth.py]: Thanks for the wording edit!

**Pull request merged**